DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -161,9 +161,15 @@ export class SearchSortFilterAndPaginate {
 
   static updateResultCount(count: number): void {
     const resultCount = <HTMLSpanElement>document.getElementById('results-count');
+    const oldResultCountMessage = resultCount.innerHTML;
     resultCount.hidden = false;
     resultCount.setAttribute('aria-hidden', 'false');
-    resultCount.textContent = count === 1 ? '1 matching result' : `${count.toString()} matching results`;
+    let newResultCountMessage = count === 1 ? '1 matching result' : `${count.toString()} matching results`;
+
+    if (newResultCountMessage === oldResultCountMessage) {
+      newResultCountMessage += '&nbsp';
+    }
+    resultCount.innerHTML = newResultCountMessage;

```

Comment:
This applies to all of the search/filter stuff where we have a results count. The results count was previously read out, but only when the content of the message (i.e. the count) changed. This change works in the same way as the other comment to make it the content "change" so it is read out when no difference in the count actually happens.

Reply:
Looks good, can we put the new stuff in a function of its own? `modifyResultCountHtmlForScreenReaderIfUnchanged` or something?